canonical/ubuntu.com
# download button fixes

## Done

Removed some unneeded styles for download buttons and gave them some bottom margin

Drive-by fixed the tabbing so ignore whitespace to review.
## QA

Click through the pages of the /download section and see that the spacing around buttons is more in line with live.
